### ISSUE TITLE
RestClient to be included in NuGet packaging

### ISSUE THREAD

**Issue by rhelms** [HotDocsCorp/hotdocs-open-sdk#35]
Will the RestClient be included in the NuGet packaging?

**Comment by RPrimrose** [comment]
Hi, this should already be included in the NuGet package. It's in the HotDocs.SDK.Cloud DLL. Are you not seeing it if you reference the NuGet package?

**Comment by rhelms** [comment]
Hi.

No. When the package was installed for the project, it only referenced HotDocs.Server and HotDocs.Server.Interop.

Regards
Reuben Helms

**Comment by rhelms** [comment]
Sorry, my mistake.

Previously, I've had a git submodule with this repo included in my project.  When I installed the NuGet package, I was thrown with the presence of HotDocs.Server and HotDocs.Server.Interop, and assumed that it had replaced HotDocs.Sdk.Server. I also did not double check the existing HotDocs.Sdk.Server to see where it was referenced from.

My apologies, and thank you for creating the NuGet package.
